### PR TITLE
Complaint Rejected and resolved pages added

### PR DESCRIPTION
**Pull request by ramakrishnaippili-eGov**

### CODE REVIEW

File: pages/ComplaintRejected.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from framework.common import Page, PageObject
+from framework.selenium_plus import *
+
+
+@PageObject
+class _ComplaintRejected(Page):
```
Comment (tarunlalwani) [inline pages/ComplaintRejected.py:6]: There are no actions on a complain rejected page?

File: pages/ComplaintResolved.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from framework.common import Page, PageObject
+from framework.selenium_plus import *
+
+
+@PageObject
+class _ComplaintResolved(Page):
+
+    def navigate(self):
```
Comment (tarunlalwani) [inline pages/ComplaintResolved.py]: No Action here also?

**Review by tarunlalwani — CHANGES_REQUESTED** [review]
Please check the 2 pages for additional actions

File: pages/ComplaintRejected.py (hunk 1 above)
Comment (ramakrishnaippili-eGov) [reply pages/ComplaintRejected.py:6]: Added reopen button with class identifier...

**Comment by ramakrishnaippili-eGov** [comment]
closing this as the structure of project changed... these changed will be added manually.

**Pull request closed without merging**